game/templates/game/stats.html

```
@@ -190,29 +190,192 @@ <h2>AI Game Summary</h2>
+            <div class="num">{{ lowest_rating }}</div>
+            <div class="label">Lowest Rating</div>
+        </div>
+
+        <div class="card">
+            <div class="num">{{ average_rating }}</div>
+            <div class="label">Average Rating</div>
+        </div>
+
+        <div class="card">
+            <div class="num">
+                {% if total_rating_change > 0 %}
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)